report/1.Introduction/1.IntroductionCtrl.tex

```diff
@@ -1,9 +1,56 @@
 % !TeX root = ../main.tex
 % This chapter is aimed to introduce both the subject of the project aswell as our motivation for the project.
 % It also aims to establish the intitial problem statement, which the analysis will be conducted based on.
+% This section should describe the basis of our project.
+% really where our solution originates, based on a real world problem,
+% even though our project is based on a "fun" project, it should still be possible to abstract it into real world applications
+
 \chapter{Introduction}
-some text
 
+Tracking airborne objects can be quite difficult.
+This difficulty has inspired different sports and hobbies, including the shooting of clay pigeons.
+Hitting a target requires the tracking of it, it's velocity and direction, as well as the calculation or prediction of its future trajectory.
+The trajectory can be impacted by number of different features, including gravity, wind, initial force, any artificial acceleration, and if the object is intelligent in anyway, the path might change procedurally.
```

Comment:
impacted by **a** number

Also maybe add punctuation after acceleration. Maybe something along the lines of "...and any artifical acceleration. Furthermore, if the object..."